app/controllers/util/FormAction.scala

```diff
@@ -42,9 +44,10 @@ object FormAction {
     }.getOrElse(Unknown)
 }
 
-case object Unknown extends FormAction
 case object Add extends FormAction
+case object Unknown extends FormAction
 case object SaveAndContinue extends FormAction
 case object SaveAndReturn extends FormAction
 case object Continue extends FormAction
 case class Remove(keys: Seq[String]) extends FormAction
+case class AddField(field: Option[String]) extends FormAction
```

Comment:
Do we ever do `AddField(None)`?

Reply:
No.  We have old Add for that.  I'll make it non-optional